contracts/mock/ProtocolInitializationTestFacet.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.9;
+
+import "../domain/BosonConstants.sol";
+import { IBosonProtocolInitializationHandler } from "../interfaces/handlers/IBosonProtocolInitializationHandler.sol";
+import { ProtocolLib } from "../protocol/libs/ProtocolLib.sol";
+import { ProtocolBase } from "../protocol/bases/ProtocolBase.sol";
+import { DiamondLib } from "../diamond/DiamondLib.sol";
+
+/**
+ * @title IBosonProtocolInitializationTestHandler
+ *
+ * @notice Mock changes to IBosonProtocolInitializationFacet.
+ *
+ */
+contract ProtocolInitializationTestFacet is IBosonProtocolInitializationHandler, ProtocolBase {
+    /**
+     * @notice Modifier to protect initializer function from being invoked twice for a given version.
+     */
+    modifier onlyUnitializedVersion(bytes32 _version) {
```

Comment:
I am sorry, but it could have been my typing too quickly. 

It should be `onlyUninitializedVersion`